v2/cmd/wails/build.go

```diff
@@ -145,16 +145,11 @@ func buildApplication(f *flags.Build) error {
 
 	// Allows cancelling the build after the first error. It would be nice if targets.Each would support funcs
 	// returning an error.
-	var targetErr error
 	targets := f.GetTargets()
-	targets.Each(func(platform string) {
-		if targetErr != nil {
-			return
-		}
-
-		if !validPlatformArch.Contains(platform) {
-			buildOptions.Logger.Println("platform '%s' is not supported - skipping. Supported platforms: %s", platform, validPlatformArch.Join(","))
-			return
+	for _, target := range targets {
+		if !validPlatformArch.Contains(target.Platform) {
+			buildOptions.Logger.Println("platform '%s' is not supported - skipping. Supported platforms: %s", target.Platform, validPlatformArch.Join(","))
```

Comment:
I think this needs a sprintf or else it will fail to work correctly with the params

Reply:
You only need 1 valid platform string right?
i.e "wails dev -platform windows" or this.
It seems to work ok so far, can you provide example that you think would be problematic?
![image](https://github.com/wailsapp/wails/assets/7006124/651ad787-9fd7-4169-b7ab-dc5d283de8cf)